src/citrine/builders/auto_configure.py

```diff
@@ -562,5 +714,31 @@ def _design_workflow_build_stage(
                 raise RuntimeError("Design workflow validation failed,"
                                    " cannot trigger design execution.")
         elif score is not None:
-            print("Triggering design execution using provided score...")
-            self.execute(score=score)
+            print("Triggering design execution...")
+            return self.execute(score=score)
+
+    def _build_default_score(self, *, objective: Objective) -> Score:
```

Comment:
I think this could be its own builder method; it seems like it could be broadly useful.